conformance/tests/httproute-invalid-cross-namespace-parent-ref.go

```diff
@@ -37,6 +39,17 @@ var HTTPRouteInvalidCrossNamespaceParentRef = suite.ConformanceTest{
 		gwNN := types.NamespacedName{Name: "same-namespace", Namespace: "gateway-conformance-infra"}
 		routeNN := types.NamespacedName{Name: "invalid-cross-namespace-parent-ref", Namespace: "gateway-conformance-web-backend"}
 
+		// The Route must have an Accepted Condition with a ParentRefNotPermitted Reason.
+		t.Run("HTTPRoute with a cross-namespace ParentRef and no ReferenceGrants allows such a reference, has an Accepted Condition with status False and Reason ParentRefNotPermitted", func(t *testing.T) {
```

Comment:
```suggestion
		t.Run("HTTPRoute with a cross-namespace ParentRef and no ReferenceGrants disallows such a reference, has an Accepted Condition with status False and Reason ParentRefNotPermitted", func(t *testing.T) {
```

Reply:
Why disallows? Isn't the `ReferenceGrant` intended to explicitly allow a reference?

Reply:
I think I was reading this wrong, my bad.

Reply:
What about this instead:

```suggestion
		t.Run("HTTPRoute with a cross-namespace ParentRef where no ReferenceGrants allows such a reference, has an Accepted Condition with status False and Reason ParentRefNotPermitted", func(t *testing.T) {
```

Reply:
yeah, it sounds better :)